starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -52,4 +54,26 @@ public void findAll() {
     List<Inventory> actualInventory = this.inventoryDAO.findAll();
     Assert.assertFalse(actualInventory.isEmpty());
   }
+
+  /**
+   * Test Create method.
+   */
+  @Test
+  public void create() {
+    Inventory inventory = new Inventory();
+    inventory.setName(NAME);
+    inventory.setProductType(PRODUCT_TYPE);
+    inventory.setId(ID);
+    Inventory outInv = this.inventoryDAO.create(inventory);
+
+    Assert.assertNotEquals(outInv.getId(), ID);
```

Comment:
What if the returned Inventory object was null? How can we check that it's not null before calling methods on it?

Great job verifying that the ids do NOT match since we want mongo to take care of handling the id generation. How can we additionally verify the total object equality for previously set fields (Id excluded)?

Reply:
Great job asserting that the object is not null. What I meant by additionally verifying the total object equality for previously set fields (besides id) was something like this:
  Assert.assertEquals(inventory.getName(), outInv.getName());  
  Assert.assertEquals(inventory.getProductType(), outInv.getProductType());